hix/project/flake.nix

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
It was unclear to me at first sight what was the difference between this file and `hix/project/flake.nix`